Rokt-Widget.podspec

```
@@ -0,0 +1,32 @@
+#
+# Be sure to run `pod lib lint rokt.podspec' to ensure this is a
+# valid spec before submitting.
+#
+# Any lines starting with a # are optional, but their use is encouraged
+# To learn more about a Podspec see https://guides.cocoapods.org/syntax/podspec.html
+#
+
+Pod::Spec.new do |s|
+  s.name             = 'Rokt-Widget'
+  s.version          = '4.9.0'
```

Current pod spec should be 4.9.1?

```suggestion
  s.version          = '4.9.1'
```

---

Done